grapher/file_grapher_test.py

```diff
@@ -0,0 +1,23 @@
+import unittest
+
+
+class TestFileGrapher(unittest.TestCase):
+    """
+    """
+    # TODO(MaikuMori): Add at least unit tests.
```

Comment:
Can we add these test cases back in? Would be nice not to regress any of these tests as part of this merge.